doc/modules/array_api.rst

```diff
@@ -0,0 +1,70 @@
+.. _array_api:
```

Comment:
We might have a problem with the TOC tree integration: the user guide TOC is displayed on:

https://output.circle-artifacts.com/output/job/861d755c-58de-4959-ad0c-423fc60fde3d/artifacts/0/doc/dispatching.html

but not anymore when navigating to the sub sections.

Reply:
Note sure if it's intentional or not. We don't seem to be very consistent at the moment.